### PR TITLE
Added address reference to CLI output

### PR DESCRIPTION
**Pull request by WatermelonArray**
This pull request just fixes the CLI bug on rojo-rbx/rojo#555

Here is a screenshot of the changes on the output
2022-06-14-130402_1920x1080_scrot

### CODE REVIEW

File: src/cli/serve.rs [hunk 1]
```diff
@@ -88,7 +88,14 @@ fn show_start_message(bind_address: IpAddr, port: u16, color: ColorChoice) -> io
     write!(&mut buffer, "Visit ")?;
 
     buffer.set_color(&green)?;
-    write!(&mut buffer, "http://localhost:{}/", port)?;
+    if bind_address.is_loopback() {
+        write!(&mut buffer, "http://localhost:")?;
```
Comment (Kampfkarren) [inline src/cli/serve.rs:92]: The same loopback check is done to display the address earlier. You would be better off storing the address in a variable that is used for both displays IMO.

**Comment by WatermelonArray** [comment]
@Kampfkarren Thanks for the feedback, all sorted!

File: src/cli/serve.rs [hunk 2]
```diff
@@ -66,6 +66,8 @@ fn show_start_message(bind_address: IpAddr, port: u16, color: ColorChoice) -> io
 
     let writer = BufferWriter::stdout(color);
     let mut buffer = writer.buffer();
+    
+    let address_string = if bind_address.is_loopback() {"localhost"} else {stringify!(bind_address)};
```
[the remark this answers is not in the document]
Comment (WatermelonArray) [reply src/cli/serve.rs]: Ah, I tried doing that before but my rust linter was saying that .to_string() wasn't a function in the struct.
Turns out my linter is slow and didn't catch up when I was trying to debug it! So I just tried using stringify!() and the problem went away - forgot that stringify!() does not do what I was trying to.

Anyway, sorted!

File: src/cli/serve.rs [hunk 3]
```diff
@@ -88,7 +90,10 @@ fn show_start_message(bind_address: IpAddr, port: u16, color: ColorChoice) -> io
     write!(&mut buffer, "Visit ")?;
 
     buffer.set_color(&green)?;
-    write!(&mut buffer, "http://localhost:{}/", port)?;
+    write!(&mut buffer, "http://{}:", address_string)?;
+
+    buffer.set_color(&green)?;
+    write!(&mut buffer, "{}/", port)?;
```
Comment (LPGhatguy) [inline src/cli/serve.rs:96]: Let's merge these calls together. You can pass multiple arguments into the same format string, something like this should work:

```rust
write!(&mut buffer, "http://{}:{}/", address_string, port)?;
```

**Review by LPGhatguy — CHANGES_REQUESTED** [review]
Hello, thanks for submitting a PR! I just got back from vacation, submitted a little feedback and approved the CI workflows to run.

**Pull request merged**